Makefile

```diff
@@ -0,0 +1,50 @@
+# Used for GDAL installation
+export PIP_INDEX_URL=https://pypi.pacificclimate.org/simple
+
+# Setup venv var
+ifeq ($(TMPDIR),)
+VENV_PATH := /tmp/rvic-daccs-venv
+else
+VENV_PATH := $(TMPDIR)/rvic-daccs-venv
+endif
+
+# Makefile Vars
+SHELL:=/bin/bash
+PYTHON=${VENV_PATH}/bin/python3
+PIP=${VENV_PATH}/bin/pip
+
+.PHONY: all
+all: install pre-commit-hook test
+
+.PHONY: apt
+apt:
+	sudo apt-get install libhdf5-serial-dev netcdf-bin libnetcdf-dev python3-setuptools
+
+.PHONY: clean-venv
+clean-venv:
+	rm -rf $(VENV_PATH)
+
+.PHONY: install
+install: venv
+	${PIP} install -U pip
+	${PIP} install -r requirements.txt -r test_requirements.txt
+	${PIP} install -e .
+
+.PHONY: install-ci
+install-ci: apt
+	pip3 install -U pip
+	pip3 install -e .[test]
+
+.PHONY: pre-commit-hook
+pre-commit-hook: venv
```

Comment:
I wonder what it does.

Reply:
The `pre-commit` hook will run black when you try to make a commit. If the code has fallen out of line with that then you can make changes. This is to prevent us from having to push "formatting" commits all the time.

Reply:
I always wondered where the code quality was checked. Good to know!